horovod/spark/lightning/remote.py

```diff
@@ -262,7 +262,7 @@ def on_epoch_end(self, trainer: "pl.Trainer", pl_module: "pl.LightningModule") -
             if hvd.rank() == 0:
                 if remote_store.saving_runs and trainer.profiler:
                     # One more file sync to push profiler result.
-                    remote_store.sync(logs_path)
+                    remote_store.sync(run_output_dir)
```

Comment:
Checkpoint is synced at every epoch.
No need to sync checkpoint files again here.
This is only for uploading profiler results, which is available at the end of training.

Reply:
Got it, will revert this change.